src/pages/help-centre/deriv-x.js

```diff
@@ -0,0 +1,184 @@
+)
+
+const DerivXMarkets = () => (
+    <ArticleWrapper>
+        <StyledHeader as="h4">{localize('What markets can I trade on Deriv X?')}</StyledHeader>
+        <Text>{/* TODO: add text here */} </Text>
+    </ArticleWrapper>
+)
+
+const DerivXTrade = () => (
+    <ArticleWrapper>
+        <StyledHeader as="h4">
+            {localize('What is the minimum and maximum amount to trade on Deriv X?')}
+        </StyledHeader>
+        <Text> {/* TODO: add trade text */} </Text>
+    </ArticleWrapper>
+)
+
+const DifferenceDMT5DTraderDerivX = () => (
+    <ArticleWrapper>
+        <StyledHeader as="h4">
+            {localize('What is the major difference between DTrader, Deriv MT5(DMT5) and Deriv X?')}
+        </StyledHeader>
+        <Text> {/* TODO: add differences text */} </Text>
+    </ArticleWrapper>
+)
+
+const DerivXAccount = () => (
+    <ArticleWrapper>
+        <StyledHeader as="h4">{localize('How do I create a deriv x account?')}</StyledHeader>
+        <Text> {/* TODO: add deriv x account text */} </Text>
+    </ArticleWrapper>
+)
+
+const DifferentAccounts = () => (
+    <ArticleWrapper>
+        <StyledHeader as="h4">
+            {localize('What are the differences between Synthetics and Financial accounts?')}
+        </StyledHeader>
+        <Text> {/* TODO: add account differences text */} </Text>
+    </ArticleWrapper>
+)
+
+const TradingPassword = () => (
+    <ArticleWrapper>
+        <StyledHeader as="h4">{localize('What is trading password?')}</StyledHeader>
+        <Text> {/* TODO: add trading password text */} </Text>
+    </ArticleWrapper>
+)
+
+const DifferentPassword = () => (
+    <ArticleWrapper>
+        <StyledHeader as="h4">
+            {localize('Why is my tarding password different from my Deriv password?')}
+        </StyledHeader>
+        <Text> {/* TODO: add password differences text */} </Text>
+    </ArticleWrapper>
+)
+
+const ResetDerivXPassowrd = () => (
+    <ArticleWrapper>
+        <StyledHeader as="h4">{localize('How do I rest my Deriv X Password?')}</StyledHeader>
+        <Text> {/* TODO: add password reset text */} </Text>
+    </ArticleWrapper>
+)
+
+const AccountInformation = () => (
+    <ArticleWrapper>
+        <StyledHeader as="h4">
+            {localize('Where can I find my Deriv X account information?')}
+        </StyledHeader>
+        <Text> {/* TODO: add account information text */} </Text>
+    </ArticleWrapper>
+)
+
+const DerivXRealMoneyAccount = () => (
+    <ArticleWrapper>
+        <StyledHeader as="h4">
+            {localize('How can I deposit funds into my Deriv X real money account?')}
+        </StyledHeader>
+        <Text> {/* TODO: add funds deposit text */} </Text>
+    </ArticleWrapper>
+)
+
+const WithdrawDerivX = () => (
+    <ArticleWrapper>
+        <StyledHeader as="h4">
+            {localize('How do I withdraw funds from my Deriv X real money account?')}
+        </StyledHeader>
+        <Text> {/* TODO: add withdraw funds text */} </Text>
```

Comment:
I think we can remove these '<Text>' tags .

Reply:
Okay, I'll clean this up